apps/android/app/src/main/java/chat/simplex/app/views/usersettings/DeveloperView.kt

```diff
@@ -0,0 +1,47 @@
+package chat.simplex.app.views.usersettings
+
+import SectionDivider
+import SectionView
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalUriHandler
+import androidx.compose.ui.res.stringResource
+import chat.simplex.app.R
+import chat.simplex.app.model.ChatModel
+import chat.simplex.app.views.TerminalView
+import chat.simplex.app.views.helpers.AppBarTitle
+
+@Composable
+fun DeveloperView(
+  m: ChatModel,
+  showCustomModal: (@Composable (ChatModel, () -> Unit) -> Unit) -> (() -> Unit),
+  withAuth: (block: () -> Unit) -> Unit
+) {
+  Column(
+    Modifier.fillMaxWidth(),
+    horizontalAlignment = Alignment.Start
```

Comment:
it's default alignment, no need to specify it by hand

Reply:
was there before, but can remove